packages/build/tests/plugins/tests.js

```diff
@@ -187,6 +188,88 @@ if (!version.startsWith('v8.')) {
   })
 }
 
+test('Deploy plugin succeeds', async t => {
+  const { socketPath, requests, stopServer } = await startDeployServer()
+  try {
+    await runFixture(t, 'empty', { flags: { buildbotServerSocket: socketPath, featureFlags: 'triggerDeploy' } })
+  } finally {
+    await stopServer()
+  }
+
+  t.true(requests.every(isValidDeployReponse))
+})
+
+test('Deploy plugin is only run with ---buildbotServerSocket', async t => {
+  const { requests, stopServer } = await startDeployServer()
+  try {
+    await runFixture(t, 'empty', { flags: { featureFlags: 'triggerDeploy' }, snapshot: false })
+  } finally {
+    await stopServer()
+  }
+
+  t.is(requests.length, 0)
+})
+
+test('Deploy plugin is only run with ---featureFlags=triggerDeploy', async t => {
```

Comment:
Same as previous comment about the test name

Reply:
See above